File: app/src/main/java/com/tobiasfried/brewkeeper/model/Ferment.java
package com.tobiasfried.brewkeeper.model;

public class Ferment {

    public Long first;
    public Long second;

    /**
     * Constructor for a Pair.
     *
     * @param first  the first object in the Pair
     * @param second the second object in the pair
     */
    public Ferment(Long first, Long second) {
        this.first = first;
        this.second = second;
    }

    /**
     * No-argument constructor for Firestore
     */
    public Ferment() {

    }
}

File: app/src/main/java/com/tobiasfried/rasa/domain/Ferment.kt
package com.tobiasfried.rasa.domain

class Ferment {

    var first: Long? = null
    var second: Long? = null

    /**
     * Constructor for a Pair.
     *
     * @param first  the first object in the Pair
     * @param second the second object in the pair
     */
    constructor(first: Long?, second: Long?) {
        this.first = first
        this.second = second
    }

    /**
     * No-argument constructor for Firestore
     */
    constructor() {

    }
}
